spot-client/src/common/css/page-layouts/more-modal.scss

```diff
@@ -0,0 +1,9 @@
+@import './../variables.scss';
+
+.more-modal {
+    background-color: var(--container-bg-color);
+    display: flex;
+    flex: 1;
+    flex-direction: column;
+    padding: 50px;
+}
```

Comment:
We should probably prioritize a modal manager to be done sooner rather than later now. It can probably have all these styles.

Reply:
I agree, actually I had issues understanding how these modals work. Let's schedule that for later. It will be painful if we don't do it, so i'm sure we won't forget